packages/zapp/console/src/components/flytegraph/ReactFlow/transformDAGToReactFlowV2.tsx

```diff
@@ -236,6 +236,11 @@ export const buildGraphMapping = (props): ReactFlowGraphMapping => {
   const parse = (props: ParseProps) => {
     const { contextNode, contextParent, rootParentNode, nodeDataProps } = props;
     let context: ReactFlowGraph | null = null;
+    if (!contextNode) {
```

Comment:
we should no longer reach this path with empty `contextNode`, but added a debugger message, just in case, for easier investigation of potential bugs in future.